CHANGELOG.md

```diff
@@ -41,6 +41,10 @@ Please add to the relevant subsections under Unreleased below on every PR where
   - NOTE: Not fixed if using `wandb.require("legacy-service")`
 - Fix Unknown image format error when uploading a gif through tensorboard. (@jacobromero in https://github.com/wandb/wandb/pull/8476)
 - Fix `OSError` from calling `Artifact.add_file` with file paths on mounted filesystems (@tonyyli-wandb in https://github.com/wandb/wandb/pull/8473)
+
+### Changed
+
+- Updated minimum version of `sentry-sdk` to 2.0.0 to address deprecation warnings. (@jacobromero in https://github.com/wandb/wandb/compare/WB-20890)
```

Comment:
you are updating a release section as well as changing another entry.
it should go to the unreleased section

Reply:
Oops this was due to pulling in the latest changes, but missed updating this